app/helpers.php

```diff
@@ -379,6 +380,11 @@ function display_regdate($user)
     return trans('users.show.joined_at', ['date' => $user->user_regdate->formatLocalized('%B %Y')]);
 }
 
+function i18n_date($datetime, $format = 'dd MMMM yyyy')
```

Comment:
if it's just date, `[IntlDateFormatter::LONG, IntlDateFormatter::NONE]` is better...

...is what I want to say but for some reason it doesn't work with `::NONE` format for time. This does work, though:

```php
$formatter = IntlDateFormatter::create(
    App::getLocale(),
    IntlDateFormatter::LONG,
    IntlDateFormatter::NONE
);
$formatter->format($datetime);
```